omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/VisitController1_9Test.java

```diff
@@ -62,16 +57,7 @@ public String getUuid() {
 	 */
 	@Override
 	public long getAllCount() {
-		return 0; //not supported
-	}
-	
-	/**
-	 * @see org.openmrs.module.webservices.rest.web.v1_0.controller.MainResourceControllerTest#shouldGetAll()
-	 */
-	@Override
-	@Test(expected = ResourceDoesNotSupportOperationException.class)
-	public void shouldGetAll() throws Exception {
-		super.shouldGetAll();
+		return 5;

```

Comment:
This shouldn't be changed, instead you need to change getAllCount() to return VisitService.getAllVisits().size()